joint_velocity_example_controller:
    type: franka_example_controllers/JointVelocityExampleController
    joint_names:
        - panda_joint1
        - panda_joint2
        - panda_joint3
        - panda_joint4
        - panda_joint5
        - panda_joint6
        - panda_joint7

joint_position_example_controller:
    type: franka_example_controllers/JointPositionExampleController
    joint_names:
        - panda_joint1
        - panda_joint2
        - panda_joint3
        - panda_joint4
        - panda_joint5
        - panda_joint6
        - panda_joint7

cartesian_pose_example_controller:
    type: franka_example_controllers/CartesianPoseExampleController
    arm_id: panda

elbow_example_controller:
    type: franka_example_controllers/ElbowExampleController
    arm_id: panda

cartesian_velocity_example_controller:
    type: franka_example_controllers/CartesianVelocityExampleController
    arm_id: panda

model_example_controller:
    type: franka_example_controllers/ModelExampleController
    arm_id: panda

force_example_controller:
    type: franka_example_controllers/ForceExampleController
    arm_id: panda
    joint_names:
        - panda_joint1
        - panda_joint2
        - panda_joint3
        - panda_joint4
        - panda_joint5
        - panda_joint6
        - panda_joint7

joint_impedance_example_controller:
    type: franka_example_controllers/JointImpedanceExampleController
    arm_id: panda
    joint_names:
        - panda_joint1
        - panda_joint2
        - panda_joint3
        - panda_joint4
        - panda_joint5
        - panda_joint6
        - panda_joint7
    k_gains:
        - 600.0
        - 600.0
        - 600.0
        - 600.0
        - 250.0
        - 150.0
        - 50.0
    d_gains:
        - 50.0
        - 50.0
        - 50.0
        - 20.0
        - 20.0
        - 20.0
        - 10.0
    radius: 0.1
    acceleration_time: 2.0
    vel_max: 0.15
    publish_rate: 10.0
    coriolis_factor: 1.0

cartesian_impedance_example_controller:
    type: franka_example_controllers/CartesianImpedanceExampleController
    arm_id: panda
    joint_names:
        - panda_joint1
        - panda_joint2
        - panda_joint3
        - panda_joint4
        - panda_joint5
        - panda_joint6
        - panda_joint7
